trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.length >= 10 ? @nameable.correct_name : @nameable.correct_name[0, 10].rstrip
```

Comment:
- [ ] I think here the logic is wrong, when the length is greater than 10, it should print only 10 word, but in your case it will print more than 10 word. your logic should be swapped.

ex:  `@nameable.correct_name.length >= 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`